Resources/public/js/views/services/plugins/ez-publishdraftplugin.js

```diff
@@ -151,28 +157,59 @@ YUI.add('ez-publishdraftplugin', function (Y) {
         },
 
         /**
+         * Creates a draft of a new content with the given fields and directly
+         * tries to publish it.
+         *
+         * @method _getServerSideFieldsErrors
+         * @param Object fieldsError the error details of each fields from the response of the REST API
+         *
+         * @protected
+         */
+        _getServerSideFieldsErrors: function (fieldsError) {
+            var serverSideFieldErrorDetails,
+                serverSideFieldsError = [];
+
+            fieldsError.forEach(function(field) {
+                field.errors.forEach(function(error) {
+                    serverSideFieldErrorDetails = new Y.eZ.FieldErrorDetails();
```

Comment:
API wise, I would organize that a bit differently to reflect the fact this object represents an error for a field definition, something like:

```
serverSideFieldErrorDetails = new Y.eZ.FieldErrorDetails({fieldDefinitionId: field._fieldTypeId});
serverSideFieldErrorDetails.parse(error);
```